### ISSUE TITLE
A11y decision or "How to use the Topbar"

### ISSUE THREAD

**Issue by ONiebsch** [holisticon/atomic-playfulness#200]
I had a discussion about a11y with myself when using the new Topbar: What parts of the Topbar should be inside a <nav> element. At least all links should be children of a <nav>, but what about the items in the right side. Can these be links too? And if so, should they be part of the same <nav> as the middle items? Should the whole Topbar be a <nav> with some <ul> inside? Maybe.

Since there are some code examples for the topbar in the documentation, I think this question is worthy being a Github issue.
